src/realtime-callbacks.js

```diff
@@ -56,6 +56,80 @@ module.exports.setNow = function(f) {
 };
 let _now = Date.now;
 
+// Internal hash of active debouncers
+const _debouncers = {};
+
+class Debouncer {
+
+    constructor(jobName) {
+        this.jobName = jobName;
+    }
+
+    complete() {
+        delete _debouncers[this.jobName];
+        this.func();
```

Comment:
Supposing the first debounce call is like `debounce('clientOffline', notifyClientOffline, 1000)` this would allow future `debounce` calls to be made with just `debounce('clientOffline')` without having to maintain a reference to the same function (but also to over-ride the callback of the `debounce` if necessary).